releases/release-4.0.13.md

```diff
@@ -0,0 +1,129 @@
+---
+title: TiDB 4.0.13 Release Notes
+---
+
+# TiDB 4.0.13 Release Notes
+
+Release date: May 28, 2021
+
+TiDB version: 4.0.13
+
+## New Feature
```

Comment:
+ TiKV

Improvements

    - Make the calculation process of store used size precise [9904](https://github.com/tikv/tikv/pull/9904)
    - Set more regions in `EpochNotMatch` to reduce region miss [9731](https://github.com/tikv/tikv/pull/9731)
    - Speed up memory freeing [10035](https://github.com/tikv/tikv/pull/10035)

Bug Fixes

    - Fix the bug that TiKV cannot startup when the end of file dict file is damaged [9963](https://github.com/tikv/tikv/pull/9963)
    - Limit CDC scan speed (128MB/s by default) [9983](https://github.com/tikv/tikv/pull/9983)
    - Reduce memory usage of CDC initial scan [10133](https://github.com/tikv/tikv/pull/10133)
    - Support back pressure CDC scan speed [10142](https://github.com/tikv/tikv/pull/10142)
    - Fix a potential OOM issue by avoiding unnecessary read for getting CDC old values [10031](https://github.com/tikv/tikv/pull/10031)
    - Fix a CDC OOM issue caused by reading old values [10197](https://github.com/tikv/tikv/pull/10197)
    - Add a timeout for s3 storage client to avoid client hangs without responses [10132](https://github.com/tikv/tikv/pull/10132)